### PR TITLE
relax pin on protobuf

### LINKED ISSUE #63: @conda-forge-admin, rerender feedstock
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Relaxing pin on protobuf per https://github.com/conda-forge/tensorflow-feedstock/issues/263#issuecomment-1236346550

We essentially rely on global pinning for libprotobuf this way.

Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #63.

Here's a checklist to do before merging.
- [x] Bump the build number if needed.

Fixes #63

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,7 +26,8 @@ requirements:
     - wheel >=0.26
     - numpy >=1.12.0
     - six >=1.10.0
-    - protobuf >=3.9.2,<3.20
+    - libprotobuf
+    - protobuf >=3.9.2
```
Comment (h-vetinari) [inline recipe/meta.yaml]:
```suggestion
    - protobuf >=3.9.2
```

**Review by h-vetinari — COMMENTED** [review]
Does this recipe really depend on the C++ library directly?

I suggest to remove this, because without a rerender, this will most likely pull in the newest builds (which are for 3.21!)

Protobuf is not part of the global pinning, so you cannot directly depend on the migrators for this.

Another option would be to add the following to `conda_build_config.yaml`
``` 
+protobuf:
+  - '3.20'
+  - '3.21'
```
which would guarantee that we have builds available for both flavours.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -46,7 +45,10 @@ test:
     - tensorboard
   commands:
     - tensorboard -h
-    - pip check
+    # ngam 20220904: pip check fails because protobuf <3.20 upstream
+    # but we want this to coexist with our libprotobuf global pin
+    # which is now at 3.20 and moving to 3.21
+    # - pip check
```
Comment (h-vetinari) [inline recipe/meta.yaml:51]: Better patch the upstream pin than remove `pip check`, IMO

Comment (h-vetinari) [reply recipe/meta.yaml]: > ugh we are just pulling a wheel here... I wonder if we should try to build from source ...

Yeah, that's not good™️. I'm guessing there could be some circular dependence with the tensorflow builds if we did it from source?

**Comment by adamjstewart** [comment]
Not only does this PR break `pip check` for the tensorboard recipe, it also [breaks `pip check`](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=565471&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d) for every package that depends on tensorboard, requiring them to either remove `pip check` or [pin protobuf](https://github.com/conda-forge/torchgeo-feedstock/pull/8/commits/53275cbf13f427354a5813d3be52d4895f66ac88). Is there any reason for this recipe to have a different set of dependency version requirements than the published wheel?

**Pull request merged**